src/helpers/SUE/sueParser.tsx

```diff
@@ -30,15 +34,23 @@ const parseSueRequests = (data) => {
       email: item?.email,
       firstName: item?.first_name,
       lastName: item?.last_name,
-      position: { longitude: item?.long, latitude: item?.lat },
+      params: {
+        details: { ...item },
```

Comment:
destructing creates a new instance and makes unnecessary process...you can pass `item` directly

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/960/commits/df11b26e53a0ba1d4d9529a1c5c15e73e36aadb8

Reply:
still not right...if you want to have item passed as details you need to pass item as details: `details: item`

Reply:
that's what i propsed in the earlier comment: https://github.com/smart-village-solutions/smart-village-app-app/pull/960#discussion_r1373411084
please read carefully and take whats there.